### ISSUE TITLE
error: required artifact is not listed by module descriptor: *#*!*.*

### ISSUE THREAD

**Issue by romac** [epfl-lara/stainless-project.g8#1]
When running sbt plugin, one receives the following report:

```
[error] a required artifact is not listed by module descriptor: *#*!*.*
```

This despite stainless running verification and the compilation and execution going through.

Reported by: @vkuncak

**Comment by romac** [comment]
This apparently stems from the comments in the `.gitignore` file of stainless. It'll be fixed in an upcoming release.

**Comment by romac** [comment]
Fixed in https://github.com/romac/stainless/commit/ced07f2ea3b9c041585b1acc168ffe7e8db5628a. Will close this issue once I release Stainless v0.2.3.

**Comment by romac** [comment]
Fixed in Stainless v0.3.0 => https://github.com/epfl-lara/stainless/releases/tag/v0.3.0
